core/session.rb

```diff
@@ -23,6 +23,7 @@
 require_relative 'commands/show_network_config_command'
```

Comment:
The Session class does not depend from this class anymore. But the new class depends. Please move the dependency too.